src/chart/treemap/TreemapSeries.ts

```diff
@@ -265,6 +266,9 @@ class TreemapSeriesModel extends SeriesModel<TreemapSeriesOption> {
                 textStyle: {
                     color: '#fff'
                 }
+            },
+            emphasis: {
+                disabled: true
```

Comment:
I think this nice new feature can be turned on by default in the 5.4

Reply:
Great